src/docfx/restore/index/DependencySlot.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Docs.Build
+{
+    internal abstract class DependencySlot
+    {
+        public string Id { get; set; }
+
+        public string Url { get; set; }
```

Comment:
Is `Url` just like `Acquirer` for internal tracking purpose? It does not need to persist on disk.

Reply:
you r right.